hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/MembershipManager.java

```diff
@@ -342,6 +346,10 @@ void updateMembers(MembersView membersView) {
 
         setMembers(MemberMap.createNew(membersView.getVersion(), members));
 
+        if (updatedLiteMember) {
+            updateMembersGroupSize();
```

Comment:
it is hard to to follow similar named methods maybe we can make `updateMemberGroupsSize()` method public and can call it here: `node.partitionService.getPartitionStateManager().updateMemberGroupsSize()`